reference/config_files/profiles.rst

```diff
@@ -470,6 +480,32 @@ Running, for instance, :command:`conan install . -pr myprofile`, the configurati
     user.myconf.build:ldflags=['--prefix prefix-value', '--flag1 value1', '--flag2 value2']
     ...
 
+.. _reference_config_files_profiles_replace_requires:
+
+[replace_requires]
+++++++++++++++++++
+
+This section allows the user to redefine requires of recipes. This can be useful when a package can be changed by a similar one like `zlib` and `zlibng`.
+It is also usefult to solve conflicts, or to replace some dependencies by system alternatives wrapped in another Conan package recipe.
+
+.. code-block:: text
+    :caption: *myprofile*
+
+    [replace_requires]
+    zlib/*: zlibng/*
```

Comment:
I think these patterns need some elaboration, the second ``*`` in the replacement will replace the value obtained in the equivalent part of the pattern.

I'd add some examples, taken from the unittests, at least the most prototypicals, and put them here.